src/main/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/sideeffects/HandEventHandler.java

```diff
@@ -15,16 +16,28 @@
 public class HandEventHandler {
 
   private final MatchRepository matchRepository;
+  private static Integer empty = null;
 
   public HandEventHandler(MatchRepository matchRepository) {
     this.matchRepository = matchRepository;
   }
 
+  @SuppressWarnings("unused")
+  @HandleBeforeSave
+  public void onBeforeSave(Hand hand) {
+    Match match = hand.getMatch();
+    boolean scoreAllowed = (match.getSumOfScoreAnnouncement() - hand.getCards().size()) == 0;
+    if (match.isLastAnnouncement() && scoreAllowed) {
+      hand.setAnnouncedScore(empty);
```

Comment:
Im Falle eines ungültigen Scores: ev. eine Fehlermeldung, dass ein ungültiger Score announced wurde?

Reply:
Unbedingt Exception mit dem richtigen code (422) werfen.
Wie siehe wahrscheinlich in HandEventHandler oder AuthController

Reply:
korrigiert.